example_configs/dex_config.yml

```diff
@@ -0,0 +1,25 @@
+# lldap configuration:
+# LLDAP_LDAP_BASE_DN:    dc=example,dc=com
+
+# ##############################
+# rest of the Dex options
+# ##############################
+
+connectors:
+  - type: ldap
+    id: ldap
+    name: LDAP
+    config:
+      host: lldap-host # make sure it does not start with `ldap://`
+      port: 3890
+      insecureNoSSL: true
```

Comment:
Maybe mention that this should be false if using LDAPS ?

Reply:
Same with the skipVerify below.

Reply:
How do we enable ldaps for lldap?

Reply:
https://github.com/nitnelave/lldap/blob/main/lldap_config.docker_template.toml#L128